src/goose/brokers/sqs/broker.clj

```diff
@@ -0,0 +1,82 @@
+(ns goose.brokers.sqs.broker
+  (:require
+   [goose.broker :as b]
+   [goose.brokers.sqs.worker :as sqs-worker]
+   [goose.brokers.sqs.sqs-requests :as sqs-request]
+   [cognitect.aws.client.api :as aws]
+   [goose.utils :as u]))
+
+(defprotocol Close
+  "Closes connection to SQS Message Broker."
+  (close [this]))
+
+(defrecord SQS [client opts]
+  b/Broker
+  ;; SQS supports nothing more than:
+  ;;   - Enqueue a job
+  ;;   - Start a worker process
+  ;;   - Enqueued jobs API
+  ;;   - Batch jobs (Will implement later)
+
+  (enqueue
+    [this job]
+    (let [{:keys [client]} this
+          {:keys [queue-url]} opts]
+
+      (sqs-request/enqueue client queue-url (u/encode-to-str job))))
+
+  (start-worker
+    [this worker-opts]
+    (let [{:keys [client]} this
+          {:keys [queue-url]} opts]
+      (sqs-worker/start client queue-url worker-opts)))
+
+  Close
+  (close [_]
+    (println "SQS client does not require explicit closure.")))
+
+(def default-opts
```

Comment:
The default-opts could point to an instance of SQS hosted locally for development purposes, for instance localstack, vsouza or roribio16's SQS docker images.

Reply:
Thank you for letting me know that this existed! 😄